modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesRuntimeProvisioningService.java

```diff
@@ -345,45 +368,44 @@ public void setRuntimeProperties(List<RuntimeProperty> runtimeProperties, Deploy
 
                 secrets.add(volume);
 
+                //create volume mount for the secretes
+                VolumeMount volumeMount = new VolumeMountBuilder()
+                        .withName(runtimeProperty.getName())
+                        .withMountPath(KubernetesPovisioningConstants.VOLUME_MOUNT_PATH + runtimeProperty.getName())
+                        .withReadOnly(true)
+                        .build();
+
+                volumeMounts.add(volumeMount);
+
                 break;
             case ENVIRONMENT:
                 if (log.isDebugEnabled()) {
                     String message = "Creating property type environment for the application:"
```

Comment:
This is fixed in following commit.
https://github.com/wso2/product-af/pull/800/commits/c707570879201754759203ff99b9c5ea5489dcba